src/test/java/com/aws/greengrass/FleetProvisioningByClaimPluginTest.java

```diff
@@ -259,6 +260,38 @@ public void GIVEN_optional_parameters_passed_to_plugin_WHEN_plugin_called_THEN_e
 
     }
 
+    @Test
+    public void GIVEN_csr_parameters_passed_to_plugin_WHEN_plugin_called_THEN_expected_methods_invoked() throws IOException, RetryableProvisioningException, InterruptedException {
```

Comment:
nit - `expected_methods_invoked` -> try to say exactly what you need to verify instead